onnxruntime/core/platform/posix/env.cc

```diff
@@ -249,6 +249,22 @@ class PosixEnv : public Env {
     return Status::OK();
   }
 
+  common::Status GetFileLength(int fd, /*out*/ size_t& file_size) const override {
```

Comment:
This function could silently discard data(because of size_t overflow), I think it is dangerous.